tests/software/pairtools/select/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+process NO_PG {
+    publishDir "${params.outdir}/nopg",
+        mode: params.publish_dir_mode
+
+    input:
+      path sam
+    output:
+      path "${sam}.filtered", emit: fil
+    script:
+      """
+      zcat < ${sam} | grep -v "@PG" > ${sam}.filtered
+      """
+}
```

Comment:
I think you can just drop this part and only test that the file exists, but I might be wrong, let's see what @drpatelh or @KevinMenden think.

Reply:
Yes I agree, if possible we'd want to avoid extra process definitions in the tests and keep them nice and simple.